biosignals.py

```diff
@@ -0,0 +1,74 @@
+import platform
+import sys
+from dataclasses import dataclass, field
+
+import plux
+
+osDic = {"Darwin": "MacOS",
+         "Linux": "Linux64",
+         "Windows": ("Win32_37", "Win64_37")}
+if platform.system() != "Windows":
+    sys.path.append("PLUX-API-Python3/{}/plux.so".format(osDic[platform.system()]))
+else:
+    if platform.architecture()[0] == '64bit':
+        sys.path.append("PLUX-API-Python3/Win64_37")
+    else:
+        sys.path.append("PLUX-API-Python3/Win32_37")
+
+
+@dataclass
+class Device:
+    """
+    Stores the integer corresponding to the port occupied by the device.
+    Defaults to port 1
+    """
+
+    source: str
+
+    def set_port(self, source: int):
+        self.source = source
+
+
+@dataclass
+class Session(Device):
+    """
+    Stores session data in a dict
```

Comment:
Glad to see you starting to add doc blocks to your classes! The cool thing about python is that you can use the `__doc__` magic method to access documentation on anything.

For example `Session.__doc__` or `Session.prepare_dict.__doc__`

Reply:
Please ellaborate on this when you can @Gevie

Reply:
Python has a lot of magic (or dunder) methods available to it by default, these are methods available to all classes and provide special operations.

An example is `num + 10` where it's actually calling a magic method on the num variable, `num.__add__(10)` which is doing the addition. You can actually extend and override the behaviour of these magic methods on classes if you wanted.

`__doc__` is one such magic method which will print out the documentation you added with your `"""` and `"""` annotation. It's good practice, but not only does it become available via a magic method, it also helps describe what you are doing to any developer reading your code without actually needing to deep dive into your code.

It's a good practice to add doc blocks like this.

Reply:
https://www.tutorialsteacher.com/python/magic-methods-in-python